tools/aapt_lite/src/main/java/com/google/devtools/build/android/BUILD.bazel

```diff
@@ -18,7 +18,7 @@ java_library(
     ],
 )
 
-kotlin_library(
+kt_jvm_library(
```

Comment:
Can we just keep `kotlin_library` as the public symbol? Updating the impl in `rules/kotlin/kotlin.bzl` will allow us to use `kotlin_library` for both bazel-common and consuming projects.